### ISSUE TITLE
Overriding GraphNode theme icons displays pitch black textures

### ISSUE THREAD

**Issue by boruok** [godotengine/godot#51493]
### Godot version

3.3.2 and 3.4.beta

### System information

Windows 10, GTS 450

### Issue description

Tried to override **graphnode** theme icon and here you go, black texture instead close button.

![Безымянный](https://user-images.githubusercontent.com/36783322/128980724-2383a7ad-efb2-4ced-9e15-cab1499932b4.png)

that happens with resizer and port too.

### Steps to reproduce

1. open attachment
2. try to change theme override -> any icon.

### Minimal reproduction project

[New Game Project.zip](https://github.com/godotengine/godot/files/6966340/New.Game.Project.zip)

**Comment by boruok** [comment]
I find out that texture turns black if contains more than 1 color. That must be documented or fixed?

**Comment by Calinou** [comment]
Can you reproduce this after disabling batching for both the project and editor in the Project Settings?

**Comment by boruok** [comment]
@Calinou yes, tried with **3.3.2** and **3.4.beta**, aslo tried version before batching merge **3.2**
